parsons/ngpvan/people.py

```diff
@@ -27,12 +75,36 @@ def find_person(self, first_name=None, last_name=None, date_of_birth=None, email
             - email_address
 
         .. note::
-            The arguments that can be passed are a selection of the possible values that
-            can be used in a search. A full list of possible values can be found
-            `here <https://developers.ngpvan.com/van-api#match-candidates>`_. To use these
-            values, pass in a dictionary using the match_map argument.
+            A full list of possible values for the json, and its structure can be found
+            `here <https://developers.ngpvan.com/van-api#match-candidates>`_.
+
+        `Args:`
+            match_json: dict
+                A dictionary of values to match against.
+            fields: The fields to return. Leave as default for all available fields
+        `Returns:`
+            A person dict
+        """
+
+        logger.info(f'Finding a match for json details.')
+
+        return self._people_search(match_json=match_json)
+
+    def update_person(self, id=None, id_type='vanid', first_name=None, last_name=None,
+                      date_of_birth=None, email=None, phone=None, phone_type=None,
+                      street_number=None, street_name=None, zip=None):
+        """
+        Update a person record based on a provided ID.
```

Comment:
I would make it clear here that the arguments provided beyond `id` are going to updated the record. That's correct, right?